api/schemas/report.json

```diff
@@ -146,6 +147,11 @@
             "type":        "boolean"
         },
 
+        "noop_pending": {
+            "description": "Whether noop resource events exist",
```

Comment:
@jtappa The Branan's description is more exact. Should I replace the description with "Whether there are changes that we decided not to apply because of noop"?